src/main/generic/utils/buffer/BufferUtils.js

```diff
@@ -4,7 +4,7 @@ class BufferUtils {
      * @return {string}
      */
     static toAscii(buffer) {
-        return String.fromCharCode.apply(null, new Uint8Array(buffer));
+        return String.fromCharCode.apply(null, BufferUtils._toUint8View(buffer));
```

Comment:
The `String.fromCharCode.apply` construct puts each individual byte of the Uint8Array as a function argument.

```js
String.fromCharCode.apply(null, [1, 2, 3, 4...])
// would get expanded to
String.fromCharCode(1, 2, 3, 4, ...)
```

Some browsers do not have enough stack space for this for long strings: https://stackoverflow.com/a/19102224

Reply:
Should I open a new issue for this, or even a `BufferUtils` helper with a more performant implementation? Or is this not an issue

Reply:
@terorie Thanks for pointing this out 👍 It's fixed in 2f08ff3d30dd6bf43403d4f4eb2df6c4cbbe6c7b.